app/ui/src/app/app.component.html

```diff
@@ -1,246 +1,243 @@
 
-        <!-- Logo -->
-        <ng-container *ngTemplateOutlet="brandTemplate"></ng-container>
-      </div>
-
-      <!-- Top Navbar -->
 
-      <nav class="collapse navbar-collapse">
-        <ul class="nav navbar-nav navbar-right navbar-iconic" role="list">
+      </div>
+      <a routerLink="/" class="navbar-brand" role="link">
+        <svg id="appLogo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1280 265"><defs><style>.cls-1{fill:#9adde8;}.cls-2{fill:#49badc;}.cls-3{fill:#fff;}</style></defs><title>syndesis_logo_full_1280px_lightbkg</title><polygon class="cls-1" points="129.95 98.77 33.99 144.38 122.22 167.37 218.18 121.75 129.95 98.77"/><polygon class="cls-2" points="218.18 121.75 218.18 121.75 122.22 167.37 242.17 198.61 218.18 121.75"/><polygon class="cls-1" points="154.99 86.86 131 10 10 67.52 129.95 98.77 154.99 86.86"/><polygon class="cls-2" points="33.99 144.38 129.95 98.77 10 67.52 10 67.52 33.99 144.38"/><polygon class="cls-2" points="189.64 25.27 131 10 154.99 86.86 213.63 102.14 189.64 25.27"/><polygon class="cls-2" points="38.54 164 62.53 240.86 121.17 256.13 121.17 256.13 97.18 179.27 97.18 179.27 38.54 164"/><polygon class="cls-1" points="242.17 198.61 122.22 167.37 97.18 179.27 121.17 256.13 242.17 198.61"/><path class="cls-3" d="M358.91,210.23c-23.57,0-43.44-17.22-44.06-41.6H329.2c.41,22.74,17.21,29.3,37.09,29.3,16.6,0,35.65-9.63,35.65-28.48,0-14.76-10.65-21.73-23.15-25.82-24-8-60.45-15.58-60.45-47.55,0-25.82,23.77-40,47.13-40,19.47,0,44.88,6.77,48.15,35.25H399.28c-.41-18.24-17-23-32.17-23-15.37,0-34.43,7.79-34.43,25.82,0,42,83.61,23,83.61,74C416.29,200.39,388.62,210.23,358.91,210.23Z"/><path class="cls-3" d="M484.32,143.22l-54.51-84h17l45.29,71.52L537.4,59.2h14.34l-54.1,83.61v64.34H484.32Z"/><path class="cls-3" d="M572.24,59.2H587.4L669,189.32h.41V59.2h13.32v148h-16.8L586,79.28h-.41V207.15H572.24Z"/><path class="cls-3" d="M760,59.2c48.56,0,76.23,25,76.23,74s-27.67,74-76.23,74H720V59.2Zm3.27,135.66c41.6,0,58.61-24.39,58.61-61.69,0-43.64-25.61-62.91-67.62-61.68h-20.9V194.86Z"/><path class="cls-3" d="M870.19,207.15V59.2H958.1V71.49H883.51v53.08h71.73v12.29H883.51v58h77.67v12.29Z"/><path class="cls-3" d="M1030.44,210.23c-23.56,0-43.44-17.22-44.05-41.6h14.34c.41,22.74,17.21,29.3,37.09,29.3,16.6,0,35.66-9.63,35.66-28.48,0-14.76-10.66-21.73-23.16-25.82-24-8-60.45-15.58-60.45-47.55,0-25.82,23.77-40,47.13-40,19.47,0,44.88,6.77,48.16,35.25h-14.35c-.41-18.24-17-23-32.17-23-15.37,0-34.43,7.79-34.43,25.82,0,42,83.61,23,83.61,74C1087.82,200.39,1060.16,210.23,1030.44,210.23Z"/><path class="cls-3" d="M1121.63,207.15V58.58H1135V207.15Z"/><path class="cls-3" d="M1212.62,210.23c-23.56,0-43.44-17.22-44.06-41.6h14.35c.41,22.74,17.21,29.3,37.09,29.3,16.6,0,35.66-9.63,35.66-28.48,0-14.76-10.66-21.73-23.16-25.82-24-8-60.45-15.58-60.45-47.55,0-25.82,23.77-40,47.13-40,19.47,0,44.88,6.77,48.16,35.25H1253c-.41-18.24-17-23-32.17-23-15.37,0-34.43,7.79-34.43,25.82,0,42,83.61,23,83.61,74C1270,200.39,1242.34,210.23,1212.62,210.23Z"/></svg>
```

Comment:
Are we all aware that this SVG needs to change based on the configuration in `config.json`?

Reply:
Hi @zregvart, do you mean the `png` file that currently exists in the `config.json` file? I see that they are different. Could you explain to me what that file does exactly?

Reply:
I think it is referring to what this logic was doing

https://github.com/syndesisio/syndesis/blob/daa24cf7791aba89340e6fb50a948c691e5e0413/app/ui/src/app/app.component.html#L9-L12

https://github.com/syndesisio/syndesis/blob/daa24cf7791aba89340e6fb50a948c691e5e0413/app/ui/src/app/app.component.ts#L110

https://github.com/syndesisio/syndesis/blob/daa24cf7791aba89340e6fb50a948c691e5e0413/app/ui/src/app/app.component.ts#L38

Do I understand this correctly? Instead of specifying this image explicitly, instead, consult with the settings to pull the correct one from config.json? Hope I didn't complicate things..

Reply:
Thank you for clearing that up @seanforyou23. I did not know. How should we proceed with the changes if we're going to use `SVG`s instead of `png` or any other conventional `img` file? Can we insert the `SVG` into the `app.component.ts` file?

Reply:
I think we can do the same type of check as before and keep using the config in the same way, just using the svg instead of png

Reply:
The only requirement that I see is that the logo and the app name needs to change. The upstream version is called Syndesis, and the productized version is called Fuse Online, the logo also changes with that. I don’t think there’s any issue in getting the productized version in SVG. Just need to make sure that the ability to swap out the branding remains the same so we don’t end up showing the community logo and branding in the product.

Reply:
@zregvart is there an automate function for this option or some form of a if/else to detect when it is Syndesis or Fuse Online? It wouldn't be efficient to load every instance of the images depending on the usage.

Reply:
Not sure how it's done when building the product. I do know that the values in `config.json` change. This is the end result:

https://github.com/syndesisio/fuse-online-install/blob/36261b43e34217ba12208c247b41bd9e80098ca3/resources/fuse-online-template.yml#L346-L355

Reply:
Hi @zregvart, I think the latest masthead commit will have the correct changes for this. @seanforyou23 also did some rework on the markup so that the `config.json` registers correctly without the oddity property values.